<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGTDemoPlanSet</class>
 <widget class="QWidget" name="QGTDemoPlanSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGTDemoPlanSet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/GTDecorationRobotTool/Resources/BaseSet_BackGround.png);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(135, 206, 250);</string>
   </property>
   <property name="text">
    <string>演示方案选择 :</string>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_DemoPlanSet_Reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>恢复默认设置</string>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_DemoPlanSet_Cancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_DemoPlanSet_Ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_DemoPlanSet_Select">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDemoPlanSet_View">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>320</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(135, 206, 250);</string>
   </property>
   <property name="text">
    <string>演示方案描述 :</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGTToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">Commen/qgttoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gtdecorationrobottool.qrc"/>
 </resources>
 <connections/>
</ui>
